pkgs/development/python-modules/flask-appbuilder/default.nix

```diff
@@ -26,23 +26,15 @@
 
 buildPythonPackage rec {
   pname = "flask-appbuilder";
-  version = "3.3.0";
+  version = "3.3.1";
 
   src = fetchPypi {
     pname = "Flask-AppBuilder";
     inherit version;
-    sha256 = "00dsfv1apl6483wy20aj91f9h5ak2casbx5vcajv2nd3i7c7v8gx";
+    sha256 = "13rlpdf3ipm39zpc62sywn8qjn6gwfbgr43x7lqpxr28br2jcg3j";
   };
 
-  patches = [
-    # https://github.com/dpgaspar/Flask-AppBuilder/pull/1610
-    (fetchpatch {
-      name = "flask_jwt_extended-and-pyjwt-patch";
-      url = "https://github.com/dpgaspar/Flask-AppBuilder/commit/7097a7b133f27c78d2b54d2a46e4a4c24478a066.patch";
-      sha256 = "sha256-ZpY8+2Hoz3z01GVtw2OIbQcsmAwa7iwilFWzgcGhY1w=";
-      includes = [ "flask_appbuilder/security/manager.py" "setup.py" ];
-    })
-  ];
+  patches = [ ./7097a7b133f27c78d2b54d2a46e4a4c24478a066.patch ];
```

Comment:
Why are we not fetching this commit? Please name it a bit better than the commit sha.